os/include/tinyara/binary_manager.h

```diff
@@ -158,6 +156,14 @@ struct kernel_binary_header_s {
 } __attribute__((__packed__));
 typedef struct kernel_binary_header_s kernel_binary_header_t;
 
+struct common_binary_header_s {
+	uint32_t crc_hash;
```

Comment:
In `user_binary_header_t`, `kernel_binary_header_t` and `common_binary_header_s`, there are common member variables.
If we can define a common structure, then we can use it. 
For example, in `binary_manager_check_user_update`.